core/forms.py

```diff
@@ -82,6 +82,7 @@ def save(self, commit=True):
             timer.stop(instance.end)
         if commit:
             instance.save()
+            self.save_m2m()
```

Comment:
:exclamation:  This one cost me a week or so. Without this line many2many relations are not being updated. It might be the case that this was a bug in BabyBuddy in general? Who knows, this makes it all work :-)

Reply:
Interesting find! This might be the first m2m relationship Baby Buddy has so it makes sense this hasn't come up before...

Reading through [Creating forms from models > the `save()` method](https://docs.djangoproject.com/en/4.0/topics/forms/modelforms/#the-save-method) it sounds like this should be done anywhere `commit=False` is used. This is the case in a few other places in the code.

Reply:
I think this will be interesting. I found this in the root form class and thought it covers most/all forms. I will try to find more instances, but not sure if I manage to find them all...